plasmapy/particles/__init__.py

```diff
@@ -46,3 +57,30 @@
 deuteron = Particle("D 1+")
 triton = Particle("T 1+")
 alpha = Particle("He-4 2+")
+
+
+def json_load_particle(json_file_object):
```

Comment:
Use the same [signature as `json.load`](https://docs.python.org/3/library/json.html#json.load).